src/iso19111/io.cpp

```diff
@@ -5214,6 +5239,26 @@ ConversionNNPtr JSONParser::buildConversion(const json &j) {
             Conversion::create(invConvProps, invMethodProps, parameters, values)
                 ->inverse()));
     }
+    else if (methodProps.getStringValue(IdentifiedObject::NAME_KEY, methodName) &&
```

Comment:
this code is exactly the same as in the WKT case ==> to be put in a utility static function